composer.json

```diff
@@ -6,27 +6,27 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.2",
-        "drupal/core": "^8.7",
-        "drupal/language_selection_page": "2.3",
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ^9",
+        "drupal/language_selection_page": "^2.6",
         "drupal/pathauto": "^1.6"
     },
     "require-dev": {
         "composer/installers": "~1.5",
         "consolidation/annotated-command": "^2.8.2",
         "consolidation/robo": "~1.4",
-        "drupal/core-composer-scaffold": "^8.8",
+        "drupal/core-composer-scaffold": "^8.9 || ^9",
         "drupal/config_devel": "~1.2",
         "drupal/devel": "~1.2",
         "drupal/drupal-extension": "~4.0",
         "drupal/redirect": "^1.4",
-        "drush/drush": "~9.0@stable",
+        "drush/drush": "~10.0",
         "guzzlehttp/guzzle": "~6.3",
```

Comment:
Not sure that "guzzlehttp/guzzle": "~6.3", is required.

Reply:
It's not a change in this PR so let's leave it there.

Reply:
Yes, it was there but I removed it in rdf_skos. https://github.com/openeuropa/rdf_skos/pull/69/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34L22
Sometimes we need to freeze specific version of the library but since we update versions of lots of libraries we need to ensure that we still need to freeze dependencies.
 "guzzlehttp/guzzle" in rdf skos was in the dependencies so it was redundant to have it also in the component's composer.json.